SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/SpatialHandoverMap.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/SpatialHandoverMap.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "SpatialGDKFunctionalTests/Public/Test2x2WorkerSettings.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestHandover/SpatialTestHandover.h"
+
+USpatialHandoverMap::USpatialHandoverMap()
+{
+	MapCategory = CI_SLOW_SPATIAL_ONLY; // This test and map is here, because I cannot run it a 100 times in a row
```

Comment:
Why can't you run it a 100 times?

Reply:
Ah, I meant that it does not pass if I run it a 100 times in a row
Seems like it does not clean up after itself, if I had to guess